src/configuration/services/varnish.md

```diff
@@ -0,0 +1,105 @@
+Varnish HTTP Proxy (alpha)
+
+Varnish is a popular HTTP proxy server, often used for caching.  It is usually not needed on Platform.sh, as each project's router provides an HTTP cache already and most more advanced use cases will use a CDN instead, both of which render Varnish redundant.
+
+However, it is possible to configure a Varnish instance as part of an application if Varnish-specific functionality is needed.
+
+## Supported versions
+
+* 5.2
+
+## How it works
+
+All incoming requests still go through the the environment's router first.  When using Varnish, a Varnish service sits between the router and the application server or servers.
+
+```
+web -> router -> varnish -> application
+                         -> application2
+```
+
+## Configuration
+
+### Add a Varnish service
+
+Add the following to your `.platform/services.yaml` file:
+
+```yaml
+varnish:
+    type: varnish:5.2
+    relationships:
+        app: 'app:http'
```

Comment:
Maybe `app: "appname:http"` to avoid confusion?

Reply:
Mm, yeah, probably wise.  Or should the relationship name be different since all of our docs and examples use `app` as the app name?  I'm leaning that way.